### PR TITLE
Adding new versions of SDKs Java and .NET in the changelog

### PR DESCRIPTION
**Pull request by delias-silva**
## Description
Adding new versions of SDKs Java and .NET in the changelog.

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/resources/changelog/current-year.es.md [hunk 1]
```diff
@@ -3,6 +3,44 @@
 Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones de Mercado Pago.
 
 ---
+## March 2021
```
Comment (lamaralmp) [inline guides/resources/changelog/current-year.es.md:6]:
```suggestion
## Marzo 2021
```

File: guides/resources/changelog/current-year.es.md [hunk 2]
```diff
@@ -3,6 +3,44 @@
...
+
+### March 4th
```
Comment (lamaralmp) [inline guides/resources/changelog/current-year.es.md:8]:
```suggestion
### 4 de marzo
```

**Review by lamaralmp — CHANGES_REQUESTED** [review]

File: guides/resources/changelog/current-year.pt.md [hunk 3]
```diff
@@ -3,6 +3,44 @@
 Descubra tudo sobre as novas versões e atualizações das integrações do Mercado Pago.
 
 ---
+## March 2021
```
Comment (lamaralmp) [inline guides/resources/changelog/current-year.pt.md:6]:
```suggestion
## Março 2021
```

File: guides/resources/changelog/current-year.pt.md [hunk 4]
```diff
@@ -3,6 +3,44 @@
...
+
+### March 4th
```
Comment (lamaralmp) [inline guides/resources/changelog/current-year.pt.md:8]:
```suggestion
### 4 de março
```

**Review by lamaralmp — CHANGES_REQUESTED** [review]

**Review by abderossi — APPROVED** [review]

**Review by lamaralmp — APPROVED** [review]

**Comment by lamaralmp** [comment]
Adicionei as alterações no PR #1083

**Pull request closed without merging**